common/src/importers/mykiCsvImporter.ts

```diff
@@ -53,13 +92,49 @@ export class MykiCsvImporter extends BaseImporter implements Importer {
         cipher.identity.city = this.getValueOrDefault(value.city);
         cipher.identity.country = this.getValueOrDefault(value.country);
         cipher.identity.postalCode = this.getValueOrDefault(value.zipCode);
+
+        this.importUnmappedFields(cipher, value, _mappedIdentityColumns);
+      } else if (value.idType !== undefined) {
+        // IdCards
+
+        cipher.identity = new IdentityView();
+        cipher.type = CipherType.Identity;
+        this.processFullName(cipher, value.idName);
+        cipher.identity.country = this.getValueOrDefault(value.idCountry);
+
+        switch (value.idType) {
+          // case "Driver's License":
+          // case "ID Card":
+          // case "Outdoor License":
+          // case "Software License":
+          // case "Tax Number":
+          // case "Bank Account":
+          // case "Insurance Card":
+          // case "Health Card":
+          // case "Membership":
+          // case "Database":
+          // case "Reward Program":
+          // case "Tour Visa":
```

Comment:
I'm guessing MyKi doesn't have these id types?

Reply:
It does have those idType but as there were no other good identity fields to map to, it uses the switch's default to map them to `cipher.identity.licenseNumber`